examples/vos0.rs

```diff
@@ -0,0 +1,49 @@
+#![deny(warnings)]
+#![deny(unsafe_code)]
+#![no_main]
+#![no_std]
+
+extern crate panic_itm;
+
+use cortex_m;
+use cortex_m_rt::entry;
+use stm32h7xx_hal::{pac, rcc, prelude::*};
+
+use cortex_m_log::println;
+use cortex_m_log::{
+    destination::Itm, printer::itm::InterruptSync as InterruptSyncItm,
+};
+
+#[entry]
+fn main() -> ! {
+    let cp = cortex_m::Peripherals::take().unwrap();
+    let dp = pac::Peripherals::take().expect("Cannot take peripherals");
+    let mut log = InterruptSyncItm::new(Itm::new(cp.ITM));
+
+    // Constrain and Freeze power
+    println!(log, "Setup PWR...                  ");
+    let pwr = dp.PWR.constrain();
+    let rcc = dp.RCC.constrain();
+    let vos = pwr.freeze_vos0(&rcc, &dp.SYSCFG);
+
+    // Constrain and Freeze clock
+    println!(log, "Setup RCC...                  ");
+    let ccdr = rcc
+        .sys_ck(480.mhz())
+        .pll1_strategy(rcc::PllConfigStrategy::Iterative)
```

Comment:
This isn't needed or explained here, should be removed

Reply:
The rcc::PllConfigStrategy::Normal strategy uses the medium range VCO which has a maximum of 420 Mhz for PLL1, and hence won't work at the 480Mhz sys_ck (it causes a panic). Iterative uses the wide range VCO which goes much higher.

Reply:
Sorry, in that case it is needed. At some point it would be nice to add another strategy that uses the wide VCO